plugins/mission/mission.cpp

```diff
@@ -83,4 +92,10 @@ void Mission::subscribe_progress(progress_callback_t callback)
     _impl->subscribe_progress(callback);
 }
 
+Mission::Result Mission::import_QGC_mission(Mission::mission_items_t &mission_items,
```

Comment:
I've little concern about this name: `import_QGC_mission()`. This has a mix of snake_case with upper case in it. I choose `QGC` over `qgc` for popularity & being catchy. But I am open to adapt the right way.
- `import_QGC_mission()` (Now)
- `import_qgc_mission()` ?
- `import_qgroundcontrol_mission()` ?
@hamishwillee @julianoes @JonasVautherin

Reply:
`import_qgroundcontrol_mission()` is good, or even `import_mission_from_qgroundcontrol()`

Reply:
OK. Thanks. I would go with `import_qgroundcontrol_mission()` as it is simple enough.